docs/cbenv.adoc

```diff
@@ -97,26 +91,26 @@ The argument for this flag is an identifier combined with a regular expression.
 So imagine you have three clusters setup with the following names:
 
 ```
-> cb-env | get cluster
-───┬────────
- 0 │ prod-us-west
- 1 │ prod-us-east
- 2 │ prod-eu-center
- 3 │ local-test
-───┴────────
+> cb-env managed | get identifier
+╭───┬────────────────╮
+│ 0 │ prod-us-east   │
+│ 1 │ prod-us-west   │
+│ 2 │ prod-eu-center │
+╰───┴────────────────╯
 ```
 
 If you wanted to run a command against all clusters in `prod-us`, you could use `--clusters prod-us.*`, e.g.
 
+[options="nowrap"]
 ```
 > buckets --clusters prod-us.*
-───┬──────────────┬───────────────┬───────────┬──────────┬──────────────────────┬───────────┬───────────────┬────────┬───────
- # │   cluster    │     name      │   type    │ replicas │ min_durability_level │ ram_quota │ flush_enabled │ status │ capella
-───┼──────────────┼───────────────┼───────────┼──────────┼──────────────────────┼───────────┼───────────────┼────────┼───────
- 0 │ prod-us-east │ default       │ couchbase │        1 │ none                 │  268.4 MB │ false         │        │ false
- 1 │ prod-us-west │ default       │ couchbase │        1 │ none                 │  268.4 MB │ false         │        │ false
- 2 │ prod-us-west │ travel-sample │ couchbase │        1 │ none                 │  209.7 MB │ false         │        │ false
-───┴──────────────┴───────────────┴───────────┴──────────┴──────────────────────┴───────────┴───────────────┴────────┴───────
+╭───┬──────────────┬───────────────┬───────────┬──────────┬──────────────────────┬───────────┬───────────────┬────────┬───────╮
+│ # │   cluster    │     name      │   type    │ replicas │ min_durability_level │ ram_quota │ flush_enabled │ status │ cloud │
```

Comment:
I suspect this row header should actually be named capella, we might have to update the command(s).